packaging/installer/kickstart.sh

```diff
@@ -1415,6 +1421,11 @@ try_package_install() {
       repo_prefix="${DISTRO_COMPAT_NAME}/${SYSVERSION}"
       ;;
     opensuse)
+      if [ "${INTERACTIVE}" = "0" ]; then
+        install_subcmd="--non-interactive --no-gpg-checks install"
+      else
+        install_subcmd="install"
```

Comment:
Thank you for your contribution @zeylos !

Since you touched this part of the code, adding `--no-gpg-checks` to `install_subcmd="install"` should act as a temporary fix for https://github.com/netdata/netdata/issues/14256 .

Reply:
I guess since you've already got 2 approvals, we can leave this one for a separate PR.

Reply:
Oh I'm so sorry I didn't read well ! I'm adding it to the interactive part as well